### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/b3524cd5aa19343686167b39872b0aad85cc2960

### CODE REVIEW

**Review by peiying2 — COMMENTED** [review]
@alexgibson  The PR is approved. Should this PR also include the `en` files so the l10n team can remove the localized versions? 

If the `en` files are removed on Bedrock side, but not triggering the changes on www-l10n repo, the sync issue is still not resolved.  If this is the case, l10n team will remove both `en` files and the corresponding localized versions.

**Comment by alexgibson** [comment]
@peiying2 https://github.com/mozilla/bedrock/issues/13811 is still an open issue, so yes for now removal will still need to be manual. Thanks

File: configs/pontoon.toml [hunk 1]
```diff
@@ -128,16 +128,6 @@ locales = [
 [[paths]]
     reference = "en/firefox/nightly/**/*.ftl"
     l10n = "{locale}/firefox/nightly/**/*.ftl"
-[[paths]]
-    reference = "en/firefox/privacy/book.ftl"
-    l10n = "{locale}/firefox/privacy/book.ftl"
-    locales = [
```
Comment (flodolo) [inline configs/pontoon.toml:134]: Note for the future: this is not particularly useful, because Smartling will translate this for all locales enabled.

**Review by flodolo — APPROVED** [review]
I'll clean up files in a separate PR (also want to update the script)

**Pull request merged**